dts/arm/ti/cc13x2_cc26x2.dtsi

```diff
@@ -149,6 +155,14 @@
 				status = "disabled";
 			};
 		};
+
+		adc0: adc@400cb008 {
+			compatible = "ti,cc13xx-cc26xx-adc";
+			reg = <0x400CB008 0x1>;
+			interrupts = <EXP_AUX_ADC 0>;
```

Comment:
just use the number, no point in having the #defines, above.   Add a comment like rtc has.